src/Microsoft.OData.Core/Microsoft.OData.Core.txt

```diff
@@ -45,6 +45,8 @@ ODataWriterCore_WriteEndCalledInInvalidState=ODataWriter.WriteEnd was called in
 
 ODataWriterCore_DeltaResourceWithoutIdOrKeyProperties=No Id or key properties were found. A resource in a delta response requires an ID or key properties be specified.
 ODataWriterCore_QueryCountInRequest=The ODataResourceSet.Count must be null for request payloads. Query counts are only supported in responses.
+ODataWriterCore_QueryNextLinkInRequest=The ODataResourceSet.NextPageLink must be null for request payloads. Next page links are only supported in responses.

```

Comment:
>ODataResourceSet [](start = 43, length = 16)

It's ODataDeltaResourceSet, Right?

Reply:
Actually, it's for either an ODataDeltaResourceSet or an ODataResourceSet. Updated wording to just refer to NextPageLink.

---
In reply to: [256646934](https://github.com/OData/odata.net/pull/1404#discussion_r256646934) [](ancestors = 256646934)